cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/change/parsing/RemovedObjectParsingStrategy.java

```diff
@@ -22,9 +26,14 @@
 
     @Override
     protected List<CfChange> doParse(Change change) {
-        return Arrays.asList(new CfRemovedObject(change.getAffectedObject().get(),
+        log.verbose("Parsing change type", change.getClass().getSimpleName(), "to custom change type",
```

Comment:
Should be `info`.

Reply:
I would recommend to use debug logging here because this log message could occur very often and does not provide valuable information to the user